### DISCUSSION TITLE
Modeling sedimentary stratas or alluvial aquifer

### DISCUSSION THREAD

**Discussion by Alfaruuki** [gempy-project/gempy#851] [General]
Please, is it possible to use gempy to create a 3d geological model of alluvial layers such that the heterogeneity is displayed in each layers of the model. 

Specifically, from the attached model output, you will see each geologic unit spread allover the surface which is not true for sedimentary stratas. 

![image](https://github.com/cgre-aachen/gempy/assets/140621039/8ec4e8dc-7090-4e4f-b06e-57c05422ae63)

I am expecting heterogeneity in the layers.  for example something like the bellow. 

![image](https://github.com/cgre-aachen/gempy/assets/140621039/283f9b8a-5a8b-4414-9a0f-4e93ee8cf6b7)

Am I missing something? please help......

**Comment by Alfaruuki** [comment]
Please, share link to videos that can help me. Thank you

**Comment by javoha** [comment]
Hello,
the models you are showing in your example have a high variation on a rather small scale. This is not necesssarily the best use case for gempys univeral co-kriging approach. I am no expert but I think there are several publications with other approaches to better represent such setttings.
What you can do: Use gempy to model bigger struccutral units and thern perform something like SGS or SIS within these units to create models similar to your example. It all depends on the amount of data you have and the purpose of your model.

**Comment by Alfaruuki** [comment]
Thank you for your response Javoha, It really meant alot to me. But what do you mean by SGS or SIS. To be more explicit about the purpose of my model. I would like to couple the resulting geological model with Flopy. Hence the variability within each layers is very important to me. Find attach for another sample. Please help. 

![image](https://github.com/cgre-aachen/gempy/assets/140621039/6cce4bd2-0c6c-418b-a5b4-5a6865b6a48b)

**Comment by javoha** [comment]
I think you should look a little more into the mathematical foundation behind gempy to be able to evaluate if it works for your use-case. SGS is Sequential Gaussian Simulation, SIS is Sequential Indicator Simulation. But whatever approach you choose really depends on your available data, purpose, etc.

**Comment by Alfaruuki** [comment]
Okay. Thank you for your response
